--- logrotate.c	2011-04-04 13:03:39.000000000 +0200
+++ logrotate.c.oden	2011-04-04 13:03:48.000000000 +0200
@@ -539,7 +539,7 @@ static int copyTruncate(char *currLog, c
     message(MESS_DEBUG, "copying %s to %s\n", currLog, saveLog);
 
     if (!debug) {
-	if ((fdcurr = open(currLog, (flags & LOG_FLAG_COPY) ? O_RDONLY : O_RDWR)) < 0) {
+	if ((fdcurr = open(currLog, (flags & LOG_FLAG_COPY) ? O_RDONLY : O_RDWR,644)) < 0) {
 	    message(MESS_ERROR, "error opening %s: %s\n", currLog,
 		    strerror(errno));
 	    return 1;
